README.md

```diff
@@ -1 +1,61 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project has the aim of practicing linters, Git/Github
+
+
+## Built With
+
+Html & Css
+
+
+## Getting Started
+
+Hello Microverse project initiate student on how to use linters, Git/Github and Github flow.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+  create .github/workflows folder in your first commit for the feature branch then add a copy of .github/workflows/linters.yml to the created folder
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] If you do not need these sections, please delete them to keep `readme` clean 😉